pending-documentation.md

```diff
@@ -1361,6 +1367,15 @@ evaluating another function, we would also be able to see the arguments of that
 function as well, unless the function was tail recursive, in which case no
 stack frame would exist once the tail call was performed.
 
+### Breaking on a set of rules or functions
```

Comment:
```suggestion
### Breaking on a set of rules or functions

```

Reply:
What's the difference here?

Reply:
Adding a newline after the level-3 header.